Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -119,8 +119,9 @@ class TopSitesPanel: UIViewController {
     //MARK: Private Helpers
     private func updateDataSourceWithSites(result: Maybe<Cursor<Site>>) {
         if let data = result.successValue {
-            self.dataSource.data = data
+            self.dataSource.topSitesCursor.child1 = data.asArray()
```

Comment:
Similar suggestion below. We should try hiding the details of whats inside the data source instead of reaching into the data source to assign topSiteCursor's child.